content/docs/0.8.1/terminology.md

```diff
@@ -24,12 +24,10 @@ Volumes in Longhorn are Kubernetes Volumes, they are created and managed as the
 
 ### Disaster Recovery Volumes
 
-A disaster recovery volume is a volume that stores data in a backup cluster in case the whole main cluster goes down. Disaster recovery volumes are used to increase the resiliency of Longhorn volumes.
-
-A disaster recovery volume can be created from a volume's backup in the backup store. And Longhorn will monitor its original backup volume and incrementally restore from the latest backup. Once the original volume in the main cluster goes down and users decide to activate the disaster recovery volume in the backup cluster, the disaster recovery volume can be activated immediately in the most condition, so it will greatly reduced the time needed to restore the data from the backup store to the volume in the backup cluster.
+A disaster recovery volume is a special volume that stores data in a backup cluster in case the whole main cluster goes down. Disaster recovery volumes are used to increase the resiliency of Longhorn volumes.
 
 ### Snapshots
 
 A **snapshot** in Longhorn is the state of a volume at a given time that is stored in the same location as the volume data on the host's physical disk. Snapshots are created instantly in Longhorn.
 
-Users can revert to any previous snapshot using the Longhorn UI. Since Longhorn is a distributed block storage system, make sure that the Longhorn volume is umounted from the host when reverting to any previous snapshot. Otherwise, Longhorn will confuse the node filesystem and cause filesystem corruption.
\ No newline at end of file
+Users can revert to any previous snapshot using the Longhorn UI. Since Longhorn is a distributed block storage system, make sure that the Longhorn volume is [umounted](https://linux.die.net/man/8/umount) from the host when reverting to any previous snapshot. Otherwise, Longhorn will confuse the node filesystem and cause filesystem corruption.
```

Comment:
Remove this paragraph since now user must use `maintenance mode` to revert to a snapshot.

Reply:
removed paragraph